doc/fuzzing.md

```diff
@@ -19,6 +19,10 @@ $ FUZZ=process_message src/test/fuzz/fuzz
 There is also a runner script to execute all fuzz targets. Refer to
 `./test/fuzz/test_runner.py --help` for more details.
 
+## Overview of Bitcoin Core fuzzing
+
+[Google](https://github.com/google/fuzzing/) has a good overview of fuzzing in general, with contributions from key architects of some of the most-used fuzzers. [This paper](https://agroce.github.io/bitcoin_report.pdf) includes an external overview of the status of Bitcoin Core fuzzing, as of summer 2021.  [John Regehr](https://blog.regehr.org/archives/1687) provides good advice on writing code that assists fuzzers in finding bugs, which is useful for developers to keep in mind.
```

Comment:
Might want to start with the link to the report, as that's the section title, move the general advice later.

Reply:
Might be a good idea; I went this way since the report assumes a good bit of fuzzing background, perhaps, and the Google link would be a better place to start from to understand it, but the ordering is a bit weird this way, maybe.